pkg/workflows/sdk/runtime.go

```diff
@@ -21,8 +21,8 @@ type FetchRequest struct {
 }
 
 type FetchResponse struct {
-	Success    bool           `json:"success"`           // true if HTTP request was successful
-	StatusCode uint8          `json:"statusCode"`        // HTTP status code
-	Headers    map[string]any `json:"headers,omitempty"` // HTTP headers
-	Body       []byte         `json:"body,omitempty"`    // HTTP response body
+	ExecutionError bool           `json:"executionError"`    // true if there were non-HTTP errors. false if HTTP request was sent regardless of status (2xx, 4xx, 5xx)
```

Comment:
@agparadiso Can you add the ErrorMessage field too? I think the other struct also includes this

Reply:
true! thanks, added